components/backstage/components/backstage_list.tsx

```diff
@@ -130,15 +131,19 @@ const BackstageList = ({searchPlaceholder = localizeMessage('backstage_list.sear
                     onClick={isFirstPage ? undefined : remainingProps.previousPage}
                     disabled={isFirstPage}
                 >
-                    <i className='icon icon-arrow-left'/>
+                    <ChevronLeftIcon>
+                        size={15}
+                    </ChevronLeftIcon>
```

Comment:
that's not going to work.
The prop needs to go inside the JSX tag:

```suggestion
                    <ChevronLeftIcon size={15}/>
```